package.json

```diff
@@ -2,19 +2,27 @@
   "name": "es-module-lexer",
   "version": "0.1.2",
   "description": "Lexes ES modules returning their import/export metadata",
-  "main": "lexer.js",
+  "main": "dist/lexer.js",
+  "module": "lexer.js",
   "scripts": {
     "test": "mocha -r esm -u tdd test/unit.js",
+    "build": "babel lexer.js --out-dir dist --plugins @babel/plugin-transform-modules-commonjs",
```

Comment:
We should compile with the `strict` option to avoid the `__esModule` interop applying.

Reply:
Done. Out of curiousity, why do you want this? It

Reply:
It was that without `strict`, I've pushed up the plugin to avoid this too.